src/monacoPatch.ts

```diff
@@ -123,12 +123,26 @@ function getAutoImport(provider: any, details: any): AutoImport | undefined {
   let specifier = '';
 
   // fortunately auto-imports are not suggested for types
-  if (details.name in XState) {
-    specifier = 'xstate';
-  } else if (details.name in XStateModel) {
-    specifier = 'xstate/lib/model';
-  } else if (details.name in XStateActions) {
+  if (
+    codeAction.description.includes('xstate/lib/actions') &&
+    details.name in XStateActions
+  ) {
     specifier = 'xstate/lib/actions';
+  } else if (
+    codeAction.description.includes('xstate/lib/actionTypes') &&
```

Comment:
any particular reason for this being added? I treat this rather as an internal module and I didn't expect it to be used externally

Reply:
the reason is that the current version of the viz shows `Auto import from ´xstate´` on three lines. But if you pick the first one it's actually an import from `xstate/lib/actionTypes`:

![CleanShot 2022-08-22 at 16 45 24](https://user-images.githubusercontent.com/167574/185950383-240e5ca9-3f7d-4b4c-b6ba-3cd0451ab15b.gif)